internal/repository/user/human_profile.go

```diff
@@ -19,8 +19,8 @@ const (
 type HumanProfileChangedEvent struct {
 	eventstore.BaseEvent `json:"-"`
 
-	FirstName         string         `json:"firstName,omitempty"`
-	LastName          string         `json:"lastName,omitempty"`
+	FirstName         *string        `json:"firstName,omitempty"`
+	LastName          *string        `json:"lastName,omitempty"`
```

Comment:
this would allow to empty first- and lastname? is this intended?

Reply:
no, i justi didnt know anymore why we did this